### PR TITLE
Implement `aflxv` and `aflx?` commands ##analysis

### LINKED ISSUE #20741: Add aflx? and some subcommands
* [ ] aflxv -> verbose with names instead of addresses
* [ ] aflxj -> json format
* [ ] aflx? -> missing help

### PR DESCRIPTION
**Pull request by miles170**
Closes #20741.

- [x] Mark this if you consider it ready to merge
- [x] I've added tests (optional)
- [ ] I wrote some lines in the [book](https://github.com/radareorg/radare2book) (optional)

**Description**

I'm not sure I implemented `aflxv` right.

### CODE REVIEW

File: libr/core/cmd_anal.c [hunk 1]
```diff
@@ -195,6 +195,15 @@ static const char *help_msg_afls[] = {
 	NULL
 };
 
+static const char *help_msg_aflx[] = {
```
Comment (trufae) [inline libr/core/cmd_anal.c:198]:
```suggestion
static RCoreHelpMessage help_msg_aflx[] = {
```

File: libr/core/cmd_anal.c [hunk 2]
```diff
@@ -195,6 +195,15 @@ static const char *help_msg_afls[] = {
...
+	"Usage:", "aflx", "[aflx] # list function xrefs",
```
Comment (trufae) [inline libr/core/cmd_anal.c:199]:
```suggestion
	"Usage:", "aflx", "[jv*] # list function xrefs",
```

File: libr/core/cmd_anal.c [hunk 3]
```diff
@@ -607,7 +616,7 @@ static const char *help_msg_afl[] = {
 	"aflq", "", "list functions in quiet mode",
 	"aflqj", "", "list functions in json quiet mode",
 	"afls", "[?asn]", "sort function list by address, size or name",
-	"aflx", "", "list function xrefs (who references or calls the current function)",
+	"aflx", "[?jv]", "list function xrefs (who references or calls the current function)",
```
Comment (trufae) [inline libr/core/cmd_anal.c:619]:
```suggestion
	"aflx", "[?*jv]", "list function xrefs (who references or calls the current function)",
```

File: libr/core/cmd_anal.c [hunk 4]
```diff
@@ -4496,36 +4505,53 @@ static int cmd_af(RCore *core, const char *input) {
...
+							continue;
 						}
-					}
-					sdb_free (db);
-					ls_free (keys);
-				}
+					} else if (rad) {                                                                                                                                                                                      r_cons_printf ("s %s;af-;af;s-\n", (const char *)kv->base.key);
+						r_cons_printf ("s %s;af-;af;s-\n", key);
+						continue;
+                    }
```
Comment (trufae) [inline libr/core/cmd_anal.c:4546]: please use tabs to indent

**Review by trufae — CHANGES_REQUESTED** [review]

**Comment by miles170** [comment]
I have no idea why the CI failed. It works locally on Ubuntu22.04.

**Comment by trufae** [comment]
Its all good. The reason why it fails is because of a race condition bug in meson that affects windows. Reported this long time ago but its still not fixed. So meh :)

thanks for the contribution! Good work!

**Pull request merged**